tools/run_tests/run_microbenchmark.py

```diff
@@ -94,7 +94,7 @@ def link(txt, tgt):
                         '--source', '%s.trace' % fnize(line), '--fmt', 'simple',
                         '--out', 'reports/%s.txt' % fnize(line)], timeout_seconds=None))
     cleanup.append(jobset.JobSpec(['rm', '%s.trace' % fnize(line)]))
-    if len(benchmarks) >= multiprocessing.cpu_count():
+    if len(benchmarks) >= min(4, multiprocessing.cpu_count()):
       jobset.run(benchmarks, maxjobs=multiprocessing.cpu_count()/2,
```

Comment:
s/`multiprocessing.cpu_count()/2`/`min(1, multiprocessing.cpu_count()/2)` for single-core machine?